packages/api/amplify_api/lib/src/graphql/paginated_model_type_impl.dart

```diff
@@ -0,0 +1 @@
+
```

Comment:
What happened here?

Reply:
This file was deleted in Travis's commit on 3/11/22 but it appears a rebase added it back again as a blank file (probably due to creating/moving of directories).  I'll delete this file.